README.md

```diff
@@ -341,6 +341,42 @@ sap.ui.define(["./a"], A => {
 
 Also refer to the `neverUseStrict` option below.
 
+### Top-Level Scripts (e.g. QUnit Testsuites)
+
+By default, modules are converted to UI5 AMD-like modules using `sap.ui.define`. In some cases, it may be necessary to include modules via script tags, such as for QUnit testsuites. Therefore, the Babel plugin supports converting modules into scripts using `sap.ui.require` instead of AMD-like modules using `sap.ui.define`. These modules can then be used as *top-level* scripts, which can be included via `<script>` tags in HTML pages.
+
+To mark a module as being converted into such a `sap.ui.require` script, you need to annotate it with the comment: `/* @sapUiRequire */` somewhere in the program code.
+
+Example:
+
+```js
+/* @sapUiRequire */
+
+// https://api.qunitjs.com/config/autostart/
+QUnit.config.autostart = false;
+
+// import all your QUnit tests here
+void Promise.all([import("unit/controller/App.qunit")]).then(() => {
+  QUnit.start();
+});
+```
+
+will be converted to:
+
+```js
+"sap.ui.require([], function () {
+  "use strict";
+
+  function __ui5_require_async(path) { /* ... */ }
+  QUnit.config.autostart = false;
+  void Promise.all([__ui5_require_async("unit/controller/App.qunit")]).then(() => {
+    QUnit.start();
+  });
+});
+```
+
+> :warning: Although `sap.ui.define` and `sap.ui.require` may appear similar from an API perspective, they have different behaviors. To understand these differences, please read the section titled "Using sap.ui.require instead of sap.ui.define on the top level" in the [Troubleshooting for Loading Modules](https://ui5.sap.com/#/topic/4363b3fe3561414ca1b030afc8cd30ce).
+
```

Comment:
Would there be a concrete, conclusive description when one is allowed to add this @sapUiRequire switch and in which cases it would be a problem? It is mentioned above when and why it is useful, but less clearly what the preconditions are.

Reply:
Therefore, the topic is described in the section "Top-Level Scripts"

Reply:
I think the explanation in the beginning is good enough